panther_docking/include/panther_docking/panther_charging_dock.hpp

```diff
@@ -140,10 +145,31 @@ class PantherChargingDock : public opennav_docking_core::ChargingDock
    */
   void getParameters(const rclcpp_lifecycle::LifecycleNode::SharedPtr & node);
 
+  /**
+   * @brief Method to update and publish the staging pose.
+   *
+   * Uses staging_x_offset_ and staging_yaw_offset_ to calculate the staging pose.
+   */
   void updateAndPublishStagingPose();
 
+  /**
+   * @brief Set the dock pose.
+   *
+   * This method sets the dock pose.  It can be used as a callback for a subscription.
+   *
+   * @param pose The dock pose.
+   */
   void setDockPose(const PoseStampedMsg::SharedPtr pose);
 
+  /**
+   * @brief Set the Wibotic info.
+   *
+   * This method sets the Wibotic info. It can be used as a callback for a subscription.
+   *
+   * @param msg The Wibotic info message.
+   */
+  void setWiboticInfo(const WiboticInfoMsg::SharedPtr msg);
```

Comment:
Please use `CamelCase` for methods name.

Reply:
That's a good question how to deal with this cause the `opennav_docking_core::ChargingDock` is written with a low first letter. `rclcpp_lifecycle::LifecycleNode` is written with `snake_case`.  

Changing our functions to `CamelCase` adds another case :D


I would leave this with a low first letter. It is okay?

Reply:
Yes you're right, let's stay in the standard of the class we inherit from